contrib/init/bitcoind.service

```diff
@@ -1,22 +1,25 @@
+# It is not recommended to modify this file in-place, because it will
+# be overwritten during package upgrades. If you want to add further
+# options or overwrite existing ones then use
+# $ systemctl edit bitcoind.service
+# See "man systemd.service" for details.
+
+# Note that almost all daemon options could be specified in
+# /etc/bitcoin/bitcoin.conf
+
 [Unit]
-Description=Bitcoin's distributed currency daemon
+Description=Bitcoin daemon
 After=network.target
 
 [Service]
+ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf -pid=/run/bitcoind/bitcoind.pid
```

Comment:
Yeah actually I take back the suggestion. Seems fine to stick with /etc/bitcoin since the config there could potentially could be shared with different bitcoin tools. And /var/bitcoind at least keeps consistency with the binary and service names.